src/rules/order.js

```diff
@@ -140,7 +140,7 @@ function isPlainRequireModule(node) {
     return false
   }
   const decl = node.declarations[0]
-  const result = (decl.id != null &&  decl.id.type === 'Identifier') &&
```

Comment:
`variableDeclarator.id` is non-null by estree spec.

Reply:
estree isn't the only parser. please include the null check, in case other parsers fail to ensure it's non-null.

Reply:
updated